packages/web-components/src/mixins/uswds-error-border.scss

```diff
@@ -8,6 +8,6 @@
 
 @media screen and (min-width: 1008px) {
     :host([error][uswds]:not([error=''])) {
-        margin-left: -1.4rem;
+        margin-left: -1.2rem;
```

Comment:
Based on @GnatalieH feedback, the margin has been changed from 14px to 12px

![Screenshot 2023-03-20 at 3 08 39 PM](https://user-images.githubusercontent.com/55560129/226442042-2d3135a7-cf54-446b-ad00-1eeeab876f4c.png)

Reply:
This will update the error state for all of the components using this mixin. Can you do a spot check to make sure they are displaying correctly? Just FYI @GnatalieH in case Sketch also needs to be updated for those too. 

<img width="467" alt="Screenshot 2023-03-20 at 4 40 00 PM" src="https://user-images.githubusercontent.com/872479/226471840-6433f3fe-4d8a-4f66-a74f-97891f54a887.png">

Reply:
Correct, I think it as mentioned on the `select` component as well. 

@GnatalieH please verify that this will apply to all v3 components on the error state

Reply:
I'm checking to confirm right now.

Reply:
@humancompanion-usds I need your feedback here. Recently you asked me to correct the spacing between the left red border and text/input field to be 12px rather than 16px. I can't remember which v3 component this was, but it was in the last sprint or two. Now, when I look at the USWDS v3 Text input and File input error states, it looks like it's 16px. Please advise so we can definitively fix all of the error states for v3 components. This is a change that will take some time to correct within the Sketch library, so please let me know as soon as possible if you can. Thank you. CC @jhonnyoddball and @jamigibbs 

<img width="976" alt="Screenshot 2023-03-21 at 9 29 42 AM" src="https://user-images.githubusercontent.com/88347129/226639204-b1d7152a-442a-4024-9968-64a8a94c0973.png">

Reply:
@humancompanion-usds has confirmed that 16px is indeed the correct margin. So please disregard the 12px value I suggested, @jamigibbs and @jhonnyoddball.